app/controllers/SubmitFormController.scala

```diff
@@ -225,18 +225,18 @@ class SubmitFormController @Inject()(mcc: MessagesControllerComponents,
                   .addingToSession(SessionKeys.returnData -> Json.toJson(sessionModel).toString())
                   .removingFromSession(SessionKeys.viewModel)
               } else {
-                logger.warn(s"[SubmitFormController][submitSuccess] Obligation end date for period $periodKey has not yet passed.")
```

Comment:
same here

Reply:
this is not an application level error, its a journey error, so i think it can be changed to warnLog. i set it as errorLog because it terminates the journey and the user is sent to a bad request page